playbook/app/pb_kits/playbook/pb_select/_select.jsx

```diff
@@ -37,6 +37,8 @@ type SelectProps = {
   id?: string,
   includeBlank?: string,
   label?: string,
+  margin: string,
+  marginBottom: string,
```

Comment:
Throws test errors if not included: `error  'marginBottom' is missing in props validation`

Reply:
But this kit does work if not included.